### ISSUE TITLE
Comments regarding software paper

### ISSUE THREAD

**Issue by magedhelmy1** [benjaminbolling/RSW#3]
3 points regarding the software paper:

First:
" State of the field: Do the authors describe how this software compares to other commonly-used packages?"

Can you please extend the paper to compare your current software with existing software that has the same/similiar functionality? For example, add a "related work" or "state of the art" section to the paper. Furthermore, please compare your current software with currently existing methods in terms pros/cons so it is clear for the user why they have to use their system over other systems.

Second:
"References: Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper citation syntax?"

I believe citation is missing from many parts of the text. For example

"....All around the world, research institutes and industrial complexes make use of workforces 6 working multiple shifts per day in order to utilise maximum efficiency and -profitability of the 7 facility...." Citation for this statement? Many of the equations used are also missing citations. Kindly make sure you have citations of all other people work in your paper and/or any claims you make in the paper-

Three:
Can you please add a high-level software architecture flow to your paper? If you check here "https://c4model.com/". A Level 3 component diagram will be highly beneficial.

**Comment by ShantanuDash** [comment]
I agree with @magedhelmy1 . Also if you could add a bit more substantial documentation and this would be good to go.
@benjaminbolling 

openjournals/joss-reviews#3431

**Comment by benjaminbolling** [comment]
@magedhelmy1 and @ShantanuDash : Thank you for your comments, I will address them and update the paper accordingly as soon as I have some time. Hopefully during this WE I will be able to push. Sorry for taking so long, I had a lot of work. :)

**Comment by benjaminbolling** [comment]
A WIP update has now been pushed. Some references left to insert.

**Comment by benjaminbolling** [comment]
@magedhelmy1 and @ShantanuDash : Please have a look when you have time.

**Comment by benjaminbolling** [comment]
And sorry for the delay.
